binaries/ofs.uio/uio/ofs_uio.py

```diff
@@ -67,11 +68,13 @@
 
 UIO_PATTERN = re.compile("uio[0-9]+")
 
+LOG = logging.getLogger(None)
+
 
 def verify_pcie_address(pcie_address):
     m = BDF_PATTERN.match(pcie_address)
     if m is None:
-        print("Invalid pcie address format{}".format(pcie_address))
+        LOG.error("Invalid pcie address format{}".format(pcie_address))
```

Comment:
When using the logging module, that docs say to prefer using the %s, %d, etc format specifiers and to append the arguments as parameters to the function call.
```python
  LOG.error("Invalid pcie address format %s", pcie_address)
```
https://docs.python.org/3/howto/logging.html#logging-variable-data